src/App.vue

```diff
@@ -21,7 +21,7 @@
   -->
 
 <template>
-	<NcSettingsSection :title="t('bruteforcesettings', 'Brute-force IP whitelist')"
+	<NcSettingsSection :name="t('bruteforcesettings', 'Brute-force IP whitelist')"
```

Comment:
this should be backported iirc? Was also a problem on older versions?

Reply:
27 no as it is still on older version but will check 28